lib/opal/nodes/args/kwoptarg.rb

```diff
@@ -27,7 +28,7 @@ def compile
 
         line "if (#{lvar_name} == null) {"
         line "  #{lvar_name} = ", expr(default_value)
-        line "}"
+        line '}'
```

Comment:
What do you think is more important in this case: readability or standardization?

Reply:
To me the human side is paramount, hence *readability*. 

I get and value standardization especially for newcomers to the language etc. but that's optimizing for writing rather than reading and I think can be solved otherwise.